### ISSUE TITLE
Abnormal Insulin like Growth Factor Binding Protein 6 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5999]
**Preferred term label:** Abnormal Insulin like Growth Factor Binding Protein 6 concentration

**Synonyms** Abnormal circulating IGFBP6

**Definition (free text, please give PubMed ID)** Concentration of Insulin like Growth Factor Binding Protein 6 in the blood is outside the normal range., PMID: 31550047

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased Insulin like Growth Factor Binding Protein 6 concentration

**Synonyms** Decreased circulating IGFBP6

**Definition (free text, please give PubMed ID)** Concentration of Insulin like Growth Factor Binding Protein 6 in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal Insulin like Growth Factor Binding Protein 6 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
No normal ranges have been determined for this protein in serum. For instance, there is no LOINC term for this test. Therefore, it is possible to do a cohort study that shows a higher level in individuals with some condition, but it is not possible to do a test in an individual and say that the level is "too high" or "too low". This means that this would not be a valid HPO term.
